stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/response/SocketErrorResponse.kt

```diff
@@ -0,0 +1,17 @@
+package io.getstream.chat.android.client.api2.model.response
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+internal data class SocketErrorResponse(
+    val error: ErrorResponse? = null,
+) {
+
+    @JsonClass(generateAdapter = true)
+    data class ErrorResponse(
+        val code: Int = -1,
+        val message: String = "",
+        val StatusCode: Int = -1,
```

Comment:
Is this capitalized on purpose?

Reply:
It is, copying the old implementation here https://github.com/GetStream/stream-chat-android/blob/81ac5f3c606237054b3ad473a35863875103ea98/stream-chat-android-client/src/main/java/io/getstream/chat/android/client/socket/ErrorResponse.kt#L8